### PR TITLE
EPMRPP-34012: fixed notification component & added Saga

### PR DESCRIPTION
**Pull request by IhorSyerkov**

### CODE REVIEW

File: app/src/controllers/notification/actionCreators.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { HIDE_NOTIFICATION, SHOW_NOTIFICATION } from './constants';
+
+/**
+ *
+ * @param message {string}
+ * @param type {('error' | 'info' | 'success')}
```
Comment (DiscoElevator) [inline app/src/controllers/notification/actionCreators.js]: I think it also makes sense to add these types as a constants

File: app/src/components/main/notification/notificationContainer/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { connect } from 'react-redux';
+import { NotificationContainer as NotificationContainerComponent } from './notificationContainer';
+
+const mapSateToProps = (state) => ({
```
Comment (DiscoElevator) [inline app/src/components/main/notification/notificationContainer/index.js]: map**State**ToProps

File: app/package.json [hunk 3]
```diff
@@ -43,8 +43,10 @@
     "react-router-dom": "^4.2.2",
     "react-select": "^1.2.1",
     "react-transition-group": "^2.3.0",
+    "recompose": "^0.26.0",
```
Comment (DiscoElevator) [inline app/package.json]: I don't see any usages in code right now. Is it necessary?

File: app/src/components/main/notification/notificationContainer/notificationContainer.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
+.notification-container {
+  position: absolute;
```
Comment (DiscoElevator) [inline app/src/components/main/notification/notificationContainer/notificationContainer.scss]: Notification should stick to the window bottom on scroll
`position: fixed`

File: app/src/routes/rootRoute.jsx [hunk 5]
```diff
@@ -19,6 +19,7 @@ import { RegistrationPage } from 'pages/outside/registrationPage';
 import { LocalizationSwitcher } from 'components/main/localizationSwitcher';
 import { authorizedRoute } from './authorizedRoute';
 import { anonymousRoute } from './anonymousRoute';
+import { Notifications } from '../components/main/notification';
```
Comment (DiscoElevator) [inline app/src/routes/rootRoute.jsx]: `from 'components/main/notification'`

File: app/src/components/main/notification/notificationContainer/notificationContainer.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+export class NotificationContainer extends PureComponent {
+  static propTypes = {
+    notifications: PropTypes.arrayOf(PropTypes.object).isRequired,
+  };
+
+  render() {
+    const { notifications } = this.props;
+
+    return (
+      <div className={styles['notification-container']}>
```
Comment (ArtsiomYeliseyenka) [inline app/src/components/main/notification/notificationContainer/notificationContainer.jsx]: Please define styles with classNames lib.

**Pull request merged**